Settings-default.toml

```diff
@@ -146,13 +146,13 @@ anonymous_access = true
 
 # Settings for Geo Engine Pro
 [user]
-user_registration = true
+registration = true
 admin_email = "admin@localhost"
 admin_password = "adminadmin"   # Initial password for the user "admin", which is created on startup
 
 [quota]
 mode = "disabled" # "disabled", "track", "check"
-# default_available_quota = 9999
+# default = 9999
```

Comment:
I think it was more clear previously. It's not clear what "default" is meant here (available quota)

Reply:
[dcfbdd1](https://github.com/geo-engine/geoengine/pull/913/commits/dcfbdd104ab32abd61d215b54ca2a848d4eb4720)